pom.xml

```diff
@@ -11,20 +11,13 @@
     <properties>
         <java.version>8</java.version>
         <scala.version>2.13</scala.version>
-        <akka.version>2.6.3+5-e7606849</akka.version>
+        <akka.version>2.6.5+101-0969e9d5</akka.version>
```

Comment:
this can be changed to 2.6.6 when we have released it, and then the snapshot repo can be removed